ONSdigital/eq-author-api
# 679  - Duplicate piping

### What is the context of this PR?
This implements:
1. Easier setup of complex questionnaires for tests
1. Duplication of piping when duplicating questionnaires and sections

### How to review 
1. Checkout author branch `551-duplicate-questionnaire`
1. Create a questionnaire with piping
1. Duplicate a questionnaire/section
1. Ensure that the piping is updated as expected:
	1. When duplicating a section - references within a section are updated but external references are not
	1. When duplicating a questionnaire - all references are updated.
